src/config/spelling_ignore_words.txt

```diff
@@ -49,9 +49,11 @@ bendell
 benseymour
 bgsound
 bidirectionality
+bkardell
```

Comment:
Looks like there are some unrelated editing changes in this PR. Does it need a sync with `main` or should they be added to an editing PR?

Reply:
Weird. Merged in `main` and pushed and seems to have resolved it.